### PR TITLE
[12.0] [MIG] base_partner_merge

### PR DESCRIPTION
**Pull request by anandkansagra**
Issue [Reference](https://github.com/OCA/partner-contact/issues/630)

### CODE REVIEW

File: base_partner_merge/views/base_partner_merge.xml [hunk 1]
```diff
@@ -0,0 +1,112 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<odoo>
+    <record model="ir.actions.act_window" id="action_partner_deduplicate">
```
Comment (tbaden) [inline base_partner_merge/views/base_partner_merge.xml]: > When declaring a record in XML:
Place id attribute before model

id should be the first attribute in record

File: base_partner_merge/views/base_partner_merge.xml [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+        <field name="name">Deduplicate Contacts</field>
+        <field name="res_model">base.partner.merge.automatic.wizard</field>
+        <field name="view_type">form</field>
+        <field name="view_mode">form</field>
+        <field name="target">new</field>
+        <field name="context">{'active_test': False}</field>
+    </record>
+
+    <record model="ir.ui.view" id="base_partner_merge_automatic_wizard_form">
```
Comment (tbaden) [inline base_partner_merge/views/base_partner_merge.xml]: same as above:
id should be the first attribute in record

**Review by tbaden — CHANGES_REQUESTED** [review]
thanks for the work. i did a code review, just some minor changes.

the record id's looked weird, but these are all id's from odoo core.

File: base_partner_merge/views/base_partner_merge.xml (hunk 2 above)
Comment (anandkansagra) [reply base_partner_merge/views/base_partner_merge.xml]: Done

**Review by tbaden — APPROVED** [review]
code review: lgtm

please squash your commits

**Review by LoisRForgeFlow — APPROVED** [review]
~Tested on runbot :+1: ~ EDIT: finally the module is not needed.

**Review by LoisRForgeFlow — CHANGES_REQUESTED** [review]
Actually, this is now completly in the core base module: https://github.com/odoo/odoo/blob/12.0/odoo/addons/base/wizard/base_partner_merge_views.xml

So no need for this module anymore.

This can be closed.

**Comment by LoisRForgeFlow** [comment]
Handled in OpenUpgrade here: https://github.com/OCA/OpenUpgrade/pull/1816

**Pull request closed without merging**